### PR TITLE
Adding .rat-excludes, readme for rat and remove unneeded files lacking copyrights.Also hive/TEST009 fix.

### PR DESCRIPTION
**Pull request by sandhyasun**

### CODE REVIEW

File: core/conn/trafci/src/org/trafodion/ci/Properties/trafciDefaultLookAndFeel.properties [hunk 1]
```diff
@@ -1,3 +1,26 @@
+#
```
Comment (arvind-narain) [inline core/conn/trafci/src/org/trafodion/ci/Properties/trafciDefaultLookAndFeel.properties]: I thought we had put this file in the exception list ? Code won't be able to handle the copyright comments

Comment (arvind-narain) [inline core/conn/trafci/src/org/trafodion/ci/Properties/trafciDefaultLookAndFeel.properties]: I'll check again on this.

**Comment by sandhyasun** [comment]
jenkins, retest

**Comment by sandhyasun** [comment]
jenkins, retest

**Comment by sandhyasun** [comment]
jenkins, retest

File: core/conn/trafci/src/org/trafodion/ci/Properties/trafciDefaultLookAndFeel.properties (hunk 1 above)
Comment (sandhyasun) [inline core/conn/trafci/src/org/trafodion/ci/Properties/trafciDefaultLookAndFeel.properties]: I think the code is working fine with the copyright. The swjdbc tests and phoenix test ran fine.

**Comment by sureshsubbiah** [comment]
Thank you. I see no issues.

File: RAT_README.txt [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+FILE           :                      NOTES
+------------------------------------------------------------------------------
+          
+          
+.gitignore  : File contains pattterns that GIT reads. Cannot have comments 
+.rat-excludes   : This file used by RAT cannot have a comment section 
+package.exclude* : List of files to exclude from packaging - cannot pargse comments.      
+eclipse.*project* : Eclipse related project file
+*.csv     :  Macro file cannot have comments.
+*/hbase-trx/hbase-protocol/* : Copied from Hbse and cannot haveApache copyrights
```
Comment (DaveBirdsall) [inline RAT_README.txt]: Seems odd... if it is HBase, shouldn't it already have copyrights?

**Comment by DaveBirdsall** [comment]
One question about one of the excluded files. Otherwise everything looks good.

File: RAT_README.txt [hunk 3]
```diff
@@ -0,0 +1,68 @@
+------------------------------------------------------------------------------
+This readme contains the list of excluded files that don't have apache headers and explanations on why they don't and cannnot have headers 
+To run the RAT report for Trafodion:
+1. Install Apache Rat. Taht will install the needed jar file. 
+2. Use command line:
```
Comment (robertamarton) [inline RAT_README.txt]: Spelling - Taht -> That

File: RAT_README.txt [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+   java -jar apache-rat-0.11.jar  -E $MY_SQROOT/../../.rat-excludes -d $MY_SQROOT/../.. > <RAT output log file>
+------------------------------------------------------------------------------
+FILE           :                      NOTES
+------------------------------------------------------------------------------
+          
+          
+.gitignore  : File contains pattterns that GIT reads. Cannot have comments 
+.rat-excludes   : This file used by RAT cannot have a comment section 
+package.exclude* : List of files to exclude from packaging - cannot pargse comments.      
+eclipse.*project* : Eclipse related project file
```
Comment (robertamarton) [inline RAT_README.txt]: Spelling - pargse -> parse

File: RAT_README.txt (hunk 2 above)
Comment (robertamarton) [inline RAT_README.txt]: Spelling Hbse -> HBase, haveApache -> have Apache

**Comment by robertamarton** [comment]
Few spelling errors but looks okay.

File: RAT_README.txt (hunk 2 above)
Comment (sandhyasun) [inline RAT_README.txt]: Corrected typo.  Rewrite the comment  that was confusing about the protobuf files. Removed 2 unneeded files that had previously been copied and weren't needed.

File: RAT_README.txt (hunk 3 above)
Comment (sandhyasun) [inline RAT_README.txt]: Done

File: RAT_README.txt (hunk 4 above)
Comment (sandhyasun) [inline RAT_README.txt]: Done

**Comment by svarnau** [comment]
Disregard latest test result. Inadvertent test was cancelled.

**Comment by DaveBirdsall** [comment]
Are we ready to commit this one yet?

**Comment by robertamarton** [comment]
I think it is ready to commit.



    Roberta



*From:* DaveBirdsall [mailto:notifications@github.com]
*Sent:* Tuesday, October 13, 2015 8:31 AM
*To:* apache/incubator-trafodion <incubator-trafodion@noreply.github.com>
*Cc:* Roberta Marton <roberta.marton@esgyn.com>
*Subject:* Re: [incubator-trafodion] Adding .rat-excludes, readme for rat
and remove unneeded files lacking copyrights.Also hive/TEST009 fix. (#115)



Are we ready to commit this one yet?

—
Reply to this email directly or view it on GitHub
<https://github.com/apache/incubator-trafodion/pull/115#issuecomment-147751312>
.

**Comment by DaveBirdsall** [comment]
I will commit this.

**Pull request merged**